lib/translator/common.h

```diff
@@ -23,6 +28,34 @@ void DebugLog(const char* format, ...);
 
 void SetDebugCallback(void (*callback)(const char*));
 
+// Max consecutive pages required by NVMe PRP list is 512
+void* AllocPages(uint16_t count);
+
+void* DeallocPages(uint16_t count);
```

Comment:
```suggestion
void* DeallocPages(void* pages, uint16_t count);
```
It's difficult to deallocate a resource without knowing which one it is.

Reply:
haha good point 😅